src/diffusers/pipelines/versatile_diffusion/modeling_text_unet.py

```diff
@@ -186,6 +186,10 @@ class conditioning with `class_embed_type` equal to `None`.
         conv_out_kernel (`int`, *optional*, default to `3`): The kernel size of `conv_out` layer.
         projection_class_embeddings_input_dim (`int`, *optional*): The dimension of the `class_labels` input when
             using the "projection" `class_embed_type`. Required when using the "projection" `class_embed_type`.
+        extra_film_condition_dim (`int`, *optional*, default to `None`):
```

Comment:
Can ignore this file! It's only changed because we run `make fix-copies` and copy the changes from `UNet2DConditionModel` to this class